sympy/core/evalf.py

```diff
@@ -1193,7 +1192,7 @@ def evalf_integral(expr: 'Integral', prec: int, options: OPT_DICT) -> TMP_RES:
     return result
 
 
-def check_convergence(numer: 'Expr', denom: 'Expr', n: 'Symbol') -> tTuple[int, Any, Any]:
+def check_convergence(numer: 'Expr', denom: 'Expr', n: 'Symbol') -> tuple[int, Any, Any]:
```

Comment:
I think that we can also change `'Expr'` to `Expr` now because we have `__future__.annotations` and `if TYPE_CHECKING`.